src/main/java/com/testsigma/sdk/OCR.java

```diff
@@ -0,0 +1,8 @@
+package com.testsigma.sdk;
+
+import java.util.List;
+
+public interface OCR {
+    public List<OCRTextPoint> extractTextFromElement(OCRImage image);
```

Comment:
Both these methods does the job, do we need 2 methods?

Reply:
Yes, It's not needed. Wanted to confirm before removing.